modules/RTC/JitsiLocalTrack.js

```diff
@@ -81,6 +81,19 @@ export default class JitsiLocalTrack extends JitsiTrack {
             this._startStreamEffect(effect);
         }
 
+        const displaySurface = videoType === VideoType.DESKTOP
+            ? track.getSettings().displaySurface
+            : null;
+
+        /**
+         * Track metadata.
+         */
+        this.metadata = {
+            timestamp: Date.now(),
+            ...displaySurface ? { displaySurface } : {}
```

Comment:
Parentheses for extra readability?

Reply:
Tried. Eslint doesn't like them, says they're unnecessary

Reply:
ah right